Sources/KlaviyoForms/KlaviyoWebView/KlaviyoWebViewController.swift

```diff
@@ -298,10 +298,12 @@ func createKlaviyoWebPreview(viewModel: KlaviyoWebViewModeling) -> UIViewControl
 #if swift(>=5.9)
 @available(iOS 17.0, *)
 #Preview("Klaviyo Form") {
-    let companyId: String = "9BX3wh" // ⬅️ use a company ID that has a live form
-    _ = klaviyoSwiftEnvironment.send(.initialize(companyId))
+    let profileData = ProfileData(
+        apiKey: "9BX3wh" // ⬅️ use a company ID that has a live form
+    )
+    _ = klaviyoSwiftEnvironment.send(.initialize(profileData.apiKey ?? ""))
```

Comment:
Lets not commit this API key to a public repo, its not a huge deal but i'd prefer to keep it out of the public eye.